### PR TITLE
Fix misnamed translations

### PR DESCRIPTION
**Pull request by lilaconlee**
### Summary

This fixes some translations that were misnamed and adds the scripts I used to fix them in to `scripts`.

I'd be fine not including the scripts if folks are opposed, I just figured they might be useful again sometime.

### Checklist

- [x] Added a changelog entry

### CODE REVIEW

File: scripts/check-missing-translations.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
```
Comment (crookedneighbor) [inline scripts/check-missing-translations.js]: Could we turn this into a test instead?

File: scripts/update-translation-key.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var fs = require('fs');
```
Comment (crookedneighbor) [inline scripts/update-translation-key.js]: Could we include a comment here that illustrates command line usage?

Comment (lilaconlee) [reply scripts/update-translation-key.js]: Think we need that in addition to the bit in `DEVELOPMENT.md`?

Comment (crookedneighbor) [reply scripts/update-translation-key.js]: Oh, I just didn't see that.

**Comment by lilaconlee** [comment]
Moved to the missing translations script into a test, pinned until we get the translations that are actually missing not just misnamed.

**Pull request merged**